change/@microsoft-teams-js-288b4aeb-d82e-48a5-9d09-73197b7f2265.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Moved subcapabilities and APIs within legacy namespace to teams namespace.",
```

Comment:
Can you put `legacy` and `teams` in backticks please? I'll be doing another pass on the changelog soon, but this will help get ahead of it.